signer/xpi/cose.go

```diff
@@ -294,35 +294,37 @@ func verifyCOSESignatures(signedFile signer.SignedFile, truststore *x509.CertPoo
 		return fmt.Errorf("xpi: cose.sig contains %d signatures, but expected %d", len(xpiSig.signMessage.Signatures), len(signOptions.COSEAlgorithms))
 	}
 
-	intermediateCerts, eeCerts, algs, err := validateCOSEMessageStructureAndGetCertsAndAlgs(xpiSig.signMessage)
+	// intermediateCerts
+	_, eeCerts, algs, err := validateCOSEMessageStructureAndGetCertsAndAlgs(xpiSig.signMessage)
 	if err != nil {
 		return fmt.Errorf("xpi: cose.sig is not a valid COSE SignMessage: %w", err)
 	}
 
-	// check that we can verify EE certs with the provided intermediates
-	intermediates := x509.NewCertPool()
-	for _, intermediateCert := range intermediateCerts {
-		intermediates.AddCert(intermediateCert)
-	}
-	cndigest := sha256.Sum256([]byte(signOptions.ID))
-	dnsName := fmt.Sprintf("%x.%x.addons.mozilla.org", cndigest[:16], cndigest[16:])
+	// // check that we can verify EE certs with the provided intermediates
```

Comment:
Why are things being commented out rather than deleted?

Reply:
Yeah I forgot to finish fixing the verification code. It requires an upstream change https://github.com/mozilla-services/pkcs7/pull/56